appveyor.yml

```diff
@@ -0,0 +1,23 @@
+# Operating system (build VM template)
+os: Visual Studio 2015
+
+install:
+  - set PATH=C:\Ruby22\bin;%PATH%
```

Comment:
Here the version of Ruby is hardcoded, so will always use the same version of Ruby, even that you have different versions in the `matrix` defined beneath.

Take a look at my sample test package gem:

https://github.com/luislavena/test-ruby-c-extension/blob/master/appveyor.yml

Please note that `make` in this case is available thanks to rake-compiler and DevKit integration.

If you need just `make`, take a look to this thread in AppVeyor:

http://help.appveyor.com/discussions/problems/2504-what-is-the-right-way-to-include-the-ruby-devkit

Hope that helps.